08-regression2.Rmd

```diff
@@ -240,14 +244,19 @@ regression, we can start to compare and contrast these methods as well as the
 predictions made by them. To start, let's look at the visualization of the
```

Comment:
Wondering if in this section (or somewhere in this chapter), we should discuss two other common issues with linear regression? Collinearity and outliers? Not in their full gory detail, but at least that the coefficients become unstable with highly correlated predictors (and recommend that they use domain expertise to remove one of the highly correlated predictors, if there were two that were highly correlated for example). K-NN doesn't suffer from this, right? So that is why I am thinking it's worth pointing out this here.

Also outliers - that can really swing the the prediction line/surface depending on where they are... We might want to at least mention this. And again, K-NN is going to be less bothered by this, I would think.

Reply:
oh, both of those are fantastic ideas. will add that now.